src/Tribe/Admin/Helpers.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+/**
+ * Class with a few helpers for the Administration Pages
+ */
+class Tribe__Events__Admin__Helpers {
+	/**
+	 * Static Singleton Holder
+	 * @var Tribe__Events__Admin__Helpers|null
+	 */
+	protected static $instance;
+
+	/**
+	 * Static Singleton Factory Method
+	 *
+	 * @return Tribe__Events__Admin__Helpers
+	 */
+	public static function instance() {
+		if ( ! isset( self::$instance ) ) {
+			$className      = __CLASS__;
+			self::$instance = new $className;
+		}
+
+		return self::$instance;
+	}
+
+
+	/**
+	 * Matcher for Admin Pages related to Post Types
+	 *
+	 * @param  boolean|string|array|null $id What will be checked to see if we return true or false
+	 *
+	 * @return boolean
+	 */
+	public function is_post_type_screen( $post_type = true ){
+		global $current_screen;
+
+		// Not in the admin we don't even care
+		if ( ! is_admin() ){
+			return false;
+		}
+
+		// Avoid Notices by checking the object type of WP_Screen
+		if ( ! is_a( $current_screen, 'WP_Screen' ) ){
+			return false;
+		}
+
+		$defaults = array(
+			Tribe__Events__Main::POSTTYPE,
+			Tribe__Events__Main::VENUE_POST_TYPE,
+			Tribe__Events__Main::ORGANIZER_POST_TYPE,
+		);
+
+		// Match any Post Type form Tribe
+		if ( true === $post_type && in_array( $current_screen->post_type, $defaults ) ){
+			return true;
+		}
+
+		// Match any of the post_types set
+		if ( ! is_scalar( $post_type ) && in_array( $current_screen->post_type, (array) $post_type ) ){
+			return true;
+		}
+
+		// Match a specific Post Type
+		if ( $current_screen->post_type === $post_type ){
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Matcher for administration pages that are from Tribe the easier way
+	 *
+	 * @param  boolean|string|array|null $id What will be checked to see if we return true or false
+	 *
+	 * @return boolean
+	 */
+	public function is_screen( $id = true ){
```

Comment:
Hrm, same comment about the default being a boolean here. I feel like I'm missing your original goal with the booleans. What do they accomplish that `null` would not?